tasks/server.js

```diff
@@ -1,7 +1,7 @@
 const express = require('express');
 
 const app = express();
-const PORT = process.env.PORT || 3000;
+const PORT = process.env.PORT || 3030;
```

Comment:
Is there a reason for this port change? It looks like Lighthouse is still trying to access `3000`. I see this flash before the Lighthouse script ends:

![image](https://user-images.githubusercontent.com/1754906/71101780-49c16700-2185-11ea-8bcb-32474bb1658e.png)

Reply:
If I locally change the port back to 3000, I get this:

```
> YOURNAMEHERE-design-system@0.0.1 lighthouse /Users/ethanmuller/git/design-system-starter
> node tasks/lighthouse.js

Error:  { id: 'aria-roles',
  title: '`[role]` values are valid',
  description:
   'ARIA roles must have valid values in order to perform their intended accessibility functions. [Learn more](https://web.dev/aria-roles/).',
  score: null,
  scoreDisplayMode: 'notApplicable',
  numericValue: undefined,
  displayValue: undefined,
  explanation: undefined,
  errorMessage: undefined,
  warnings: undefined,
  details: undefined }
(node:45406) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'items' of undefined
    at audits.forEach.audit (/Users/ethanmuller/git/design-system-starter/tasks/lighthouse.js:81:52)
    at Array.forEach (<anonymous>)
    at launchChromeAndRunLighthouse.then.results (/Users/ethanmuller/git/design-system-starter/tasks/lighthouse.js:78:12)
(node:45406) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:45406) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Are you getting the same result? I can't tell if this is the script failing, or if it's Lighthouse doing its job.